server/executor/trigger/traceid.go

```diff
@@ -0,0 +1,66 @@
+package trigger
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubeshop/tracetest/server/expression"
+	"github.com/kubeshop/tracetest/server/model"
+)
+
+func TRACEID() Triggerer {
+	return &traceidTriggerer{}
+}
+
+type traceidTriggerer struct{}
+
+func (t *traceidTriggerer) Trigger(ctx context.Context, test model.Test, opts *TriggerOptions) (Response, error) {
+	response := Response{
+		Result: model.TriggerResult{
+			Type: t.Type(),
+		},
+	}
+
+	return response, nil
+}
+
+func (t *traceidTriggerer) Type() model.TriggerType {
+	return model.TriggerTypeTRACEID
+}
+
+func (t *traceidTriggerer) Resolve(ctx context.Context, test model.Test, opts *TriggerOptions) (model.Test, error) {
+	traceid := test.ServiceUnderTest.TRACEID
+
+	if traceid == nil {
+		return test, fmt.Errorf("no settings provided for TRACEID triggerer")
+	}
```

Comment:
Should we test here if traceid.Id is empty too? Something like:

```suggestion
	if traceid == nil || traceid.Id == "" {
		return test, fmt.Errorf("no settings provided for TRACEID triggerer")
	}
```